liota/device_comms/EtherNetIP_DeviceComms.py

```diff
@@ -0,0 +1,95 @@
+# -*- coding: utf-8 -*-

```

Comment:
Use Pep 8 convention for naming the module. https://www.python.org/dev/peps/pep-0008/#package-and-module-names
Not only for this module and class, but for all of your code.